extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -15,6 +15,10 @@ define('ext.wikia.adEngine.lookup.lookupFactory', [
 	'use strict';
 
 	function create(module) {
+		if (adContext.get('bidders.prebidAE3')) {
+			return null;
```

Comment:
Is it safe to return null?

Reply:
This module shouldn't be called and we want to preserve memory and cut all logic here. Also, in case I forgot about some places in code to reroute, Emil will get a clear error at this place.